### PR TITLE
RFC 048: Concepts work plan

### PR DESCRIPTION
**Pull request by jamieparkinson**
Rendered version: https://github.com/wellcomecollection/docs/blob/rfc-concepts-work/rfcs/048-concepts-rfcs/README.md

### CODE REVIEW

File: rfcs/048-concepts-rfcs/README.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+We need to design an API that will allow us to populate concept pages. This may be one API for all concept types (ie subjects or people), or different APIs may provide different types. It should be designed in such a way that the future inclusion of curated content is possible. There should be sufficient information on a concept to disambiguate it (eg birth/death dates for people).
+
+#### 2. Knowledge graph population
+
+We need to populate a knowledge graph (wherein the nodes are concepts or have an injective mapping to concepts) from arbitrary sources eg. LCSH, LC names, MeSH, Wikidata, Wikipedia. This will likely initially be just one source without consideration of edges.
+
+#### 3. Identified concepts on works
+
+Where works are tagged with identifiable (in practice, Library of Congress) concepts, we need to be able to match these up with concepts in our knowledge graph. The outcome here is that, given a work, we should be able to know its subjects/agents/etc, and given a concept, we should be able to query for the works that are tagged with it.
```
Comment (harrisonpim) [inline rfcs/048-concepts-rfcs/README.md:40]: > (in practice, Library of Congress)

MeSH too!

Comment (jamieparkinson) [reply rfcs/048-concepts-rfcs/README.md:40]: Oops yes good point

File: rfcs/048-concepts-rfcs/README.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# RFC 048: Concepts work plan
+
+**Status:** :construction: Draft :building_construction:
+
+**Last updated:** 29/04/2022
+
+## Context
+
+We are now starting work on introducing "concepts" to the Wellcome digital platform: identifiable entities like subjects, people, organisations, genres, etc. Works can be tagged with concepts, and concepts can be linked to other concepts by (for example) predicates, in this way forming a knowledge graph. There is a future possibility to infer a work's subjects, as well as manual tagging.
```
Comment (alexwlchan) [inline rfcs/048-concepts-rfcs/README.md:9]: minor: I would reword this last sentence to be more explicit, so we're all on the same page.

> We will start by using concepts that are manually tagged in the source catalogues, and in future we might infer a work's concepts.

File: rfcs/048-concepts-rfcs/README.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+This RFC outlines the major technical decisions that will need to be made around data modelling, API design, and system architecture in order to serve this outcome - it does not aim to _make_ any of these decisions.
+
+## What needs to be done
+
+This approach presupposes the following architectural decisions:
+
+- There is a separate API endpoint within the catalogue for concepts 
+- There is a knowledge graph of concepts that exists independently of works
+- Concepts that works are tagged with are identifiable (the those identifiers exist in the graph)
```
Comment (alexwlchan) [inline rfcs/048-concepts-rfcs/README.md:28]: not sure what you're getting at with the parenthetical

**Review by alexwlchan — APPROVED** [review]
LGTM.

File: rfcs/048-concepts-rfcs/README.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+We need to populate a knowledge graph (wherein the nodes are concepts or have an injective mapping to concepts) from arbitrary sources eg. LCSH, LC names, MeSH, Wikidata, Wikipedia. This will likely initially be just one source without consideration of edges.
+
+#### 3. Identified concepts on works
+
+Where works are tagged with identifiable (in practice, Library of Congress and MeSH) concepts, we need to be able to match these up with concepts in our knowledge graph. The outcome here is that, given a work, we should be able to know its subjects/agents/etc, and given a concept, we should be able to query for the works that are tagged with it.
+
+## Extra thoughts
+
+- Is there any level of denormalisation of concepts onto works / vice versa?
+- How will we handle multiple source identifiers mapping to individual canonical identifiers?
```
Comment (harrisonpim) [inline rfcs/048-concepts-rfcs/README.md:45]: NB we have an existing structure/process for handling this in the prototype graph - happy to chat about that elsewhere if beyond the scope of this RFC

File: rfcs/048-concepts-rfcs/README.md (hunk 3 above)
Comment (jamieparkinson) [reply rfcs/048-concepts-rfcs/README.md:28]: Nor am I

File: rfcs/048-concepts-rfcs/README.md (hunk 4 above)
Comment (jamieparkinson) [reply rfcs/048-concepts-rfcs/README.md:45]: Will be worth talking about when the 3rd step is RFC'd - suspect it's going to involve a service separate to both pipelines
